tempesta_fw/http.c

```diff
@@ -1645,6 +1674,9 @@ tfw_http_msg_create_sibling(TfwHttpMsg *hm, struct sk_buff **skb,
 	if (unlikely(!shm))
 		return NULL;
 
+	/* New message created, so maybe it should be in whitelist. */
+	tfw_http_mark_wl_new_msg(hm->conn, shm, *skb);
```

Comment:
This a pipelined request case. originally previous message and the new one were in the same skb, so the skb mark is the same for both. No need to start the search once again: just copy whitelist flag from the previous message.

Reply:
Corrected.